pkg/meta/model/db.go

```diff
@@ -22,11 +22,18 @@ import (
 
 // DBInfo provides meta data describing a DB.
 type DBInfo struct {
-	ID         int64     `json:"id"`      // Database ID
-	Name       ast.CIStr `json:"db_name"` // DB name.
-	Charset    string    `json:"charset"`
-	Collate    string    `json:"collate"`
-	Deprecated struct {  // Tables is not set in infoschema v2, use infoschema SchemaTableInfos() instead.
+	ID      int64     `json:"id"`      // Database ID
+	Name    ast.CIStr `json:"db_name"` // DB name.
+	Charset string    `json:"charset"`
+	Collate string    `json:"collate"`
+	// Database-level active-active and soft-delete configuration
+	// Empty string means not set, "ON"/"OFF" for boolean values
+	IsActiveActive        string   `json:"is_active_active,omitempty"`         // "ON"/"OFF" or ""
```

Comment:
I think it's better to keep the same with TableInfo, only add two fields:

```
	// IsActiveActive indicates if the table has active-active replication enabled
	IsActiveActive bool `json:"is_active_active,omitempty"`

	// SoftdeleteInfo holds the soft delete configuration for this table
	SoftdeleteInfo *SoftdeleteInfo `json:"softdelete_info,omitempty"`
```

Reply:
It is intended.

When overriding from db options, you have to known that it is set\unset. This is a real problem for options that did not default to `OFF`, like `SOFTDELET_JOB_ENABLE`. You can not tell it is set by user to `false` or user have not set anything. The former, you need inheritance, the latter, you dont want.

And if there is only one `*SoftdeleteInfo`, and it is not nil, it becomes more obsecure. Does not that  mean set by user to these fields? Or only some fields are set?